docusaurus/docs/plugins/plugins-getting-started.md

```diff
@@ -1,30 +1,20 @@
 # Getting Started
 
-## Use Case: Rancher Dashboard Development
+This guide will walk through creating a new extension from scratch.
 
-The goal is to not complicate the development process for Rancher. Although the code has moved around and even once we pull out code into the `pkg` folder, the development
-process for Rancher will be the same as it is.
+## Prerequisites
 
-You should be able to use `yarn dev` as before - the UI will build and run just as it did pre-2.6.5.
+You will need a newer version of nodejs installed.
```

Comment:
If we lack a specific version, maybe `need a recent version` would be better?

Reply:
Done